walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -31,24 +33,51 @@
 
     // Find the collection of time range to hold the meeting
     Collection<String> requiredAttendees = request.getAttendees();
-    Collection<TimeRange> queries = new ArrayList<>();
-    int preStartTime = TimeRange.START_OF_DAY;
+    Collection<String> optionalAttendees = request.getOptionalAttendees();
+    // If there is no required attendees, then consider only the optinal attendees
+    requiredAttendees = requiredAttendees.isEmpty() ? optionalAttendees : requiredAttendees;
+
+    Collection<TimeRange> requiredAttendeesQuery = new ArrayList<>();
+    Collection<TimeRange> optionalAttendeesQuery = new ArrayList<>();
```

Comment:
consider giving more concrete names?
```requiredAttendeesQuery``` sounds like the time range only for required attendees.

Reply:
Fixed!